BoardSupport.md

```diff
@@ -0,0 +1,77 @@
+# Notes on Specific Boards
+
+## RP2040 / Raspberry Pi Pico
+
+First of all add support for RP2040 boards in the Arduino IDE by going to Boards Manager within the Tools menu.
+Enter "RP2040" in the search box, select "Arduino Mbed OS RP2040 Boards" and install the latest version (2.3.1 at the time of writing).
+
+ConfigurableFirmata v2.10.1 requires a minor modification to the `Boards.h` file in the `src/utility` folder for the Pico to work with the Firmata libraries.
+On macOS by default this file can be found here: `~/Documents/Arduino/libraries/ConfigurableFirmata/src/utility/Boards.h`.
+
+Add the following prior to the `#else` instruction towards the end of this file to enable support for the Raspberry Pi Pico:
```

Comment:
I'm a bit confused now. After we merge this PR, this information is obsolete, right?

Reply:
Yes, simply added in case anyone reads this and would like to use the current release on a Pico. This should be removed once the next version is released, I'm not sure on the timelines - days, months or years.

Reply:
Ok, got it. Neither do I, actually. I haven't looked into how to generate a new release at all.